en/setup/dev_env_linux_ubuntu.md

```diff
@@ -1,304 +1,80 @@
-## Setup keys
-sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
-sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
-## For keyserver connection problems substitute hkp://pgp.mit.edu:80 or hkp://keyserver.ubuntu.com:80 above.
-sudo apt-get update
-## Get ROS/Gazebo
-sudo apt install ros-melodic-desktop-full -y
-## Initialize rosdep
-sudo rosdep init
-rosdep update
-## Setup environment variables
-rossource="source /opt/ros/melodic/setup.bash"
-if grep -Fxq "$rossource" ~/.bashrc; then echo ROS setup.bash already in .bashrc;
-else echo "$rossource" >> ~/.bashrc; fi
-eval $rossource
-## Install rosinstall and other dependencies
-sudo apt install python-rosinstall build-essential -y
-```
-
-Install the [MAVROS \(MAVLink on ROS\)](../ros/mavros_installation.md) package.
-This enables MAVLink communication between computers running ROS, MAVLink enabled autopilots, and MAVLink enabled GCS.
-
-> **Tip** MAVROS can be installed as an Ubuntu package or from source.
-  Source is recommended for developers.
-
-
-```sh
-## Create catkin workspace (ROS build system)
-mkdir -p ~/catkin_ws/src
-cd ~/catkin_ws
-
-## Install dependencies
-sudo apt-get install python-wstool python-rosinstall-generator python-catkin-tools -y
-
-## Initialise wstool
-wstool init ~/catkin_ws/src
-
-## Build MAVROS
-### Get source (upstream - released)
-rosinstall_generator --upstream mavros | tee /tmp/mavros.rosinstall
-### Get latest released MAVLink package
-rosinstall_generator mavlink | tee -a /tmp/mavros.rosinstall
-### Setup workspace & install deps
-wstool merge -t src /tmp/mavros.rosinstall
-wstool update -t src
-rosdep install --from-paths src --ignore-src --rosdistro melodic -y
-```
-
-> **Note** If you use a Ubuntu-based distro and the command `rosdep install --from-paths src --ignore-src --rosdistro melodic -y` fails, you can try to force the command to run by executing `rosdep install --from-paths src --ignore-src --rosdistro melodic -y --os ubuntu:bionic`
-
-```sh
-## Build!
-catkin build
-## Re-source environment to reflect new packages/build environment
-catkin_ws_source="source ~/catkin_ws/devel/setup.bash"
-if grep -Fxq "$catkin_ws_source" ~/.bashrc; then echo ROS catkin_ws setup.bash already in .bashrc;
-else echo "$catkin_ws_source" >> ~/.bashrc; fi
-source ~/.bashrc
-```
-
-
-## NuttX-based Hardware
-
-Install the following dependencies to build for NuttX based hardware: Pixhawk, Pixfalcon, Pixracer, Pixhawk 3, Intel® Aero Ready to Fly Drone.
-
-> **Note** Packages with specified versions should be installed with the specified package version.
-
-```sh
-sudo apt-get install python-serial openocd \
-    flex bison libncurses5-dev autoconf texinfo \
-    libftdi-dev libtool zlib1g-dev -y
-```
-
 
+<!-- Do we need to add to our scripts or can we assume correct version installs over?
```

Comment:
@dagar original instructions suggested removing old version of arm-none-eabi compiler before installing new one. We don't do that in script. Can you confirm it is not needed?

Reply:
Generally I'm against removing stuff on someones computer. I think it's one thing to install dependencies but I disagree with removing them.

That being said by adding the custom arm-none-eabi compiler and then adding the path **to the beginning** we prioritize our custom compiler anyway and it should work just fine even if you have it already installed.

Reply:
Makes sense! Thanks. Another good reason to remove our old script and instructions :-)